policies/read_accounts.json

```diff
@@ -0,0 +1,11 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [{
+        "Sid": "ReadAccounts",
+        "Effect": "Allow",
+        "Action": [
+            "vaultadmin:GetAccountInfo"
```

Comment:
Do we support this action for non root users today? I don't see it in the actionMaps...